source/_integrations/rfxtrx.markdown

```diff
@@ -8,6 +8,7 @@ ha_category:
   - Switch
   - Binary Sensor
   - Sensor
+ha_iot_class: Local Polling
```

Comment:
It is push actually, the values are instantly read from the serial port when it is pushed in.

```suggestion
ha_iot_class: Local Push
```

Reply:
I checked with another similar serial sensor, don't know which one, corrected.